lxd/db/operationtype/operation_type.go

```diff
@@ -244,6 +247,8 @@ func (t Type) Permission() string {
 		return "manage-containers"
 	case InstanceDelete:
 		return "manage-containers"
+	case InstanceRebuild:
+		return "operate-containers"
```

Comment:
@stgraber are you happy with this permission level for instance rebuild? Should it be operate or manage?